.gitignore

```diff
@@ -68,3 +68,8 @@ test.py
 
 .env
 *.ini
+MUSICALBUMS
+MUSICARTISTS
+MOVIES
+SHOWS
+MUSICPLAYLISTS
```

Comment:
Ah, sorry, I always end up using the Node.js slot generator because I need to test it anyway.  Forgot the python slot generator creates these files.  Nevermind then :)